cmd/networkcmd/start.go

```diff
@@ -149,6 +153,32 @@ func StartNetwork(*cobra.Command, []string) error {
 		ux.PrintTableEndpoints(resp.ClusterInfo)
 	}
 
+	relayerStoredConfigPath := filepath.Join(app.GetAWMRelayerSnapshotConfsDir(), snapshotName+jsonExt)
```

Comment:
nit: assign `var snapshotFileName` `snapshotName+jsonExt` to make code more readable. in this case you don't need jsonExt constant as it's used only once as far as I can see

Reply:
will just keep it as is